plugins/copr.py

```diff
@@ -148,6 +148,8 @@ def run(self):
 
         repo_filename = "{}/_copr_{}-{}.repo" \
                         .format(self.base.conf.get_reposdir, copr_username, copr_projectname)
+        modules_repo_filename = "/etc/fm.modules.d/_copr_{}-{}.cfg" \
```

Comment:
Perhaps this could apply? https://fedoraproject.org/wiki/Packaging:Guidelines#The_directory_is_owned_by_a_package_which_is_not_required_for_your_package_to_function. It would be multiple ownership then.